package.json

```diff
@@ -30,6 +30,7 @@
     "@types/fs-extra": "5.0.5",
     "@types/jest": "24.0.12",
     "@types/lodash": "4.14.123",
+    "@types/node": "^12.6.2",
```

Comment:
Doest't block this PR but in this or or separately, we can pin deps like this and `rimraf`. Maybe use green keeper or something (in future) to always pin deps. I miss this all the time myself as the defaults (override-able via `.npmrc` I think) don't pin versions by default.

Reply:
Good catch, updated :+1: